src/index.ts

```diff
@@ -356,8 +356,8 @@ const glitchElement = (element: HTMLElement, layers: LayerDefinition[], options:
     let container: HTMLDivElement;
     if (! alreadyGlitched) {
         container = document.createElement('div');
-        container.style.display = getComputedStyle(element).getPropertyValue('display');
```

Comment:
Removing this breaks the ability to glitch elements inline, but it was already not working before as elements were incorrectly stacked.